test/utils/test-utils.js

```diff
@@ -248,6 +248,10 @@ const runInfo = (args, testPath) => {
     return run(testPath, ['info'].concat(args), false);
 };
 
+const windowsPathConverter = (path) => {
+    return path.replace(/\\/g, '\\\\');
```

Comment:
Never do there, you have `path.normalize`, replacing `\` on `//` is not a safe in some cases